app/src/main/java/org/oppia/app/customview/interaction/EditTextBindingAdapters.java

```diff
@@ -0,0 +1,13 @@
+package org.oppia.app.customview.interaction;
+
+import android.text.TextWatcher;
+import android.widget.EditText;
+import androidx.databinding.BindingAdapter;
+/** This bindingAdapter tells the compiler how to set a TextWatcher on an EditText. */
```

Comment:
Nit: newline needed above the comment. It's a bit crowded as-is.

Reply:
Done